src/en/administration/install_docker.md

```diff
@@ -44,6 +49,28 @@ sudo mv nginx.conf /etc/nginx/sites-enabled/lemmy.conf
 
 You will also need to setup TLS, for example with [Let's Encrypt](https://letsencrypt.org/). After this you need to restart Nginx to reload the config.
 
+## Cerbot on NGINX
+
+View: [https://certbot.eff.org/instructions](https://certbot.eff.org/instructions)
+
+Install Certbot on Debian/Ubuntu: 
+```
+sudo apt install certbot nginx 
+```
+
+then run:
+
+```
+sudo systemctl stop nginx
+sudo certbot certonly --standalone --post-hook "systemctl restart nginx"
```

Comment:
You dont need to stop nginx if you run certbot with --nginx, instead of --standalone. Though you need to install python3-certbot-nginx then. Its what we are doing in the ansible playbook.

https://github.com/LemmyNet/lemmy-ansible/blob/main/lemmy.yml#L46

Reply:
I also don't think this should be here, because the instructions are different for lots of distros. That's why just linking to the site is better.